### ISSUE TITLE
After updating to 0.3-beta.28 hlint doesn't work.

### ISSUE THREAD

**Issue by flatlizard** [carymrobbins/intellij-haskforce#244]
After updating to 0.3-beta.28 hlint doesn't work. Every time hlint running it ends up with notification:
```
	hlint
	Nonzero exit status (1) from command: C:\...\cabal\bin\hlint.exe --json  C:/.../Filename.hs
	Process stderr: 
	Caused by: null
	Configure
```
Project reconfiguring doesn't help.

**Comment by carymrobbins** [comment]
@flatlizard - If you run that same command on the command line, what's the output?  After running it, you may want to also run `echo %errorlevel%` (I believe) to see the exit status.

**Comment by narinari** [comment]
I occur same problem.The exit status is "1".

OS: OS X El Capitan
GHC: 7.10.3
HLint: 1.9.26

**Comment by develop7** [comment]
+1 here

**Comment by carymrobbins** [comment]
This should be resolved in the latest release, **v0.3-beta.29**.  Thanks for the report!

**Comment by develop7** [comment]
@carymrobbins that was really fast, thanks!
